samples/crypto/test_sample/README.rst

```diff
@@ -0,0 +1,258 @@
+|                    | EdDSA       | `RFC 8032 - Edwards-Curve Digital Signature Algorithm (EdDSA)`_            | `RFC 8032 - Edwards-Curve Digital Signature Algorithm (EdDSA)`_            |
++--------------------+-------------+----------------------------------------------------------------------------+----------------------------------------------------------------------------+
+| Hash               | SHA256      | `NIST - SHA`_                                                              | `NIST - SHA`_                                                              |
++--------------------+-------------+----------------------------------------------------------------------------+----------------------------------------------------------------------------+
+|                    | SHA512      |                                                                            | `NIST - SHA`_                                                              |
++--------------------+-------------+----------------------------------------------------------------------------+----------------------------------------------------------------------------+
+| HMAC               | HMAC SHA256 | `NIST - HMAC`_                                                             | `NIST - HMAC`_                                                             |
++--------------------+-------------+----------------------------------------------------------------------------+----------------------------------------------------------------------------+
+|                    | HMAC SHA512 |                                                                            | `NIST - HMAC`_                                                             |
++--------------------+-------------+----------------------------------------------------------------------------+----------------------------------------------------------------------------+
+| HKDF               | HKDF SHA256 | `RFC 5869 - HMAC-based Extract-and-Expand Key Derivation Function (HKDF)`_ | `RFC 5869 - HMAC-based Extract-and-Expand Key Derivation Function (HKDF)`_ |
++--------------------+-------------+----------------------------------------------------------------------------+----------------------------------------------------------------------------+
+|                    | HKDF SHA512 |                                                                            | `RFC 5869 - HMAC-based Extract-and-Expand Key Derivation Function (HKDF)`_ |
++--------------------+-------------+----------------------------------------------------------------------------+----------------------------------------------------------------------------+
+
+
+Requirements
+************
+
+* One of the following development boards:
+
+  * |nRF9160DK|
+  * |nRF5340DK|
+  * |nRF52840DK|
+  * |nRF52DK|
+  * |nRF51DK|
+
+* something else??
+
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/bluetooth/central_bas`
+
```

Comment:
Another paragraph (sub-paragraph?) needs to be added to explain the configuration used to override ztest default log formatting. 



"
ztest custom formatting

The crypto test application replaces the standard ztest formatting to provide more efficient reporting of running tests and test results. This is done by setting the configuration `CONFIG_ZTEST_TC_UTIL_USER_OVERRIDE` which enables the use of ./include_override/tc_util_user_override.h

Setting this config will replace the ztest macros `TC_START` and  `Z_TC_END_RESULT` with versions more suited for reporting results for cryptographic tests.
"

Reply:
done